runtimes/politest/src/lib.rs

```diff
@@ -607,11 +722,168 @@ impl pallet_preimage::Config for Runtime {
+	type Currency = Balances;
+	type DelegationBondLessDelay = DelegationBondLessDelay;
+	type LeaveCandidatesDelay = LeaveCandidatesDelay;
+	type LeaveDelegatorsDelay = LeaveDelegatorsDelay;
+	type MaxBottomDelegationsPerCandidate = MaxBottomDelegationsPerCandidate;
+	type MaxDelegationsPerDelegator = MaxDelegationsPerDelegator;
+	type MaxTopDelegationsPerCandidate = MaxTopDelegationsPerCandidate;
+	type MinBlocksPerRound = MinBlocksPerRound;
+	type MinCandidateStk = MinCandidateStk;
+	type MinDelegation = MinDelegation;
+	type MinDelegatorStk = MinDelegatorStk;
+	type MinSelectedCandidates = MinSelectedCandidates;
+	type MonetaryGovernanceOrigin = frame_system::EnsureRoot<AccountId>;
+	type OnCollatorPayout = ();
+	type OnNewRound = ();
+	type PayMaster = PayMaster;
+	// We use the default implementation, so we leave () here.
+	type PayoutCollatorReward = ();
+	type RevokeDelegationDelay = RevokeDelegationDelay;
+	type RewardPaymentDelay = RewardPaymentDelay;
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeHoldReason = RuntimeHoldReason;
+	type WeightInfo = pallet_parachain_staking::weights::SubstrateWeight<Runtime>;
+}
+
+impl pallet_membership::Config<pallet_membership::Instance1> for Runtime {
+	type AddOrigin = EnsureRoot<AccountId>;
+	type MaxMembers = ConstU32<50>;
+	type MembershipChanged = Oracle;
+	type MembershipInitialized = ();
+	type PrimeOrigin = EnsureRoot<AccountId>;
+	type RemoveOrigin = EnsureRoot<AccountId>;
+	type ResetOrigin = EnsureRoot<AccountId>;
+	type RuntimeEvent = RuntimeEvent;
+	type SwapOrigin = EnsureRoot<AccountId>;
+	type WeightInfo = ();
+}
+
+parameter_types! {
+	pub const MinimumCount: u32 = 3;
+	pub const ExpiresIn: Moment = 1000 * 60; // 1 mins
+	pub const MaxHasDispatchedSize: u32 = 20;
+	pub RootOperatorAccountId: AccountId = AccountId::from([0xffu8; 32]);
+	pub const MaxFeedValues: u32 = 4; // max 4 values allowd to feed in one call (USDT, USDC, DOT, PLMC).
+}
+
+impl orml_oracle::Config for Runtime {
+	type CombineData = orml_oracle::DefaultCombineData<Runtime, MinimumCount, ExpiresIn, ()>;
+	type MaxFeedValues = MaxFeedValues;
+	type MaxHasDispatchedSize = MaxHasDispatchedSize;
+	type Members = OracleProvidersMembership;
+	type OnNewData = ();
+	type OracleKey = AssetId;
+	type OracleValue = Price;
+	type RootOperatorAccountId = RootOperatorAccountId;
+	type RuntimeEvent = RuntimeEvent;
+	type Time = Timestamp;
+	// TODO Add weight info
+	type WeightInfo = ();
+}
+
+pub struct AssetPriceConverter;
+impl Convert<(AssetName, FixedU128), (AssetId, Price)> for AssetPriceConverter {
+	fn convert((asset, price): (AssetName, FixedU128)) -> (AssetId, Price) {
+		match asset {
+			AssetName::DOT => (AcceptedFundingAsset::DOT.to_assethub_id(), price),
+			AssetName::USDC => (AcceptedFundingAsset::USDC.to_assethub_id(), price),
+			AssetName::USDT => (AcceptedFundingAsset::USDT.to_assethub_id(), price),
+			AssetName::PLMC => (pallet_funding::PLMC_FOREIGN_ID, price),
+		}
```

Comment:
We use hardcoded values in the polimec runtime. Add these changes there too

Reply:
Addressed in https://github.com/Polimec/polimec-node/pull/212/commits/926710c467fffdc39e53590037ee1361e15f8955